src/components/campaignLandingPage/template.test.tsx

```diff
@@ -535,3 +582,130 @@ describe('CampaignLandingPage->EventsPanel', () => {
     expect(screen.queryByTestId('events-panel')).not.toBeInTheDocument()
   })
 })
+
+describe('CampaignLandingPage->FaqPanel', () => {
+  beforeEach(async () => {
+    await render(<FaqPanel {...(mockBaseProps as CampaignLandingPageProps)} />)
+  })
+
+  test('displays correct header and content', () => {
+    expect(screen.getByText('Reusable container header')).toBeInTheDocument()
+    expect(screen.getByText('Reusable container answer')).toBeInTheDocument()
+  })
+
+  test('displays correct default header when not specified', async () => {
+    await cleanup()
```

Comment:
I didn't know what this was doing until I looked it up. It looks like a decent portion of these tests call this cleanup function and then re-render a new one. Would it be more efficient to allow each test to be in charge of its own component render instead?

Reply:
@pwolfert Chris left similar feedback in another PR. I went ahead and removed all the `beforeEach` renders and made each individual test do it's own render.

There are still some `cleanup` calls where explicitly rendering multiple times to test both true/false states. Imo, it would make the tests less contextually clear if those were separated (`is X true` vs. `is X not false` make sense together when the context is 'X'), but could see some sensible arguments to the contrary, so happy to adapt here as well.